test/ModalSpec.tsx

```diff
@@ -1,18 +1,17 @@
-import { mount } from 'enzyme';
 import * as React from 'react';
+import { fireEvent, render, cleanup } from '@testing-library/react';
 import ModalManager from '@restart/ui/ModalManager';
-import Modal from '../src/Modal';
+import Modal, { ModalProps } from '../src/Modal';
 
 describe('<Modal>', () => {
   afterEach(() => {
-    // make sure the dangling portal elements get cleaned up
-    document.body.innerHTML = '';
+    cleanup();
```

Comment:
is this `afterEach` still needed? https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
As defined in the docs, It is recommended to use cleanup method so preferred it.
`Failing to call cleanup when you've called render could result in a memory leak and tests which are not "idempotent" (which can lead to difficult to debug errors in your tests).`
https://testing-library.com/docs/react-testing-library/api/#cleanup

Reply:
My understanding is that this should be done automatically.  Was this not working?

> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.

Reply:
I missed it. Will remove the after each hooks.